src/global_sensitivity/sobol_sensitivity.jl

```diff
@@ -107,8 +107,38 @@ function gsa(f, method::Sobol, A::AbstractMatrix, B::AbstractMatrix;
     SobolResult(Sᵢ, nothing, 2 in method.order ? Sᵢⱼ : nothing, nothing, Tᵢ, nothing)
 end
 
+function gsa(f,method::Sobol,A::AbstractVector{<:AbstractMatrix},B::AbstractVector{<:AbstractMatrix};kwargs...)
+    d,n = size(A[1])
+    res = map(A,B) do A_,B_
+        Threads.@spawn gsa(f, method, A_, B_; kwargs...)
```

Comment:
I think we should just make bigger design matrices and do the call to the user. If they want parallelism, they will already be using the batch form and will already have parallelism more optimally implemented.